tests/recipes/test_alpaca_generate.py

```diff
@@ -39,5 +31,5 @@ def test_alpaca_generate(self, tmpdir, monkeypatch):
         cmd += model_config
 
         monkeypatch.setattr(sys, "argv", cmd)
-        with pytest.raises(SystemExit):
+        with pytest.raises(SystemExit, match=""):
```

Comment:
Why is this needed?

Reply:
See comment above. This ensures the SystemExit was a successfully run command, rather than an error.